rest/aggs/aggs.go

```diff
@@ -19,11 +19,11 @@ type Client struct {
 	client.Client
 }
 
-// Get retrieves aggregate bars for a specified ticker over a given date range in custom time window sizes.
+// GetAggs retrieves aggregate bars for a specified ticker over a given date range in custom time window sizes.
 // For example, if timespan = ‘minute’ and multiplier = ‘5’ then 5-minute bars will be returned.
-func (ac *Client) Get(ctx context.Context, params GetParams, opts ...client.Option) (*AggsResponse, error) {
+func (ac *Client) GetAggs(ctx context.Context, params GetAggsParams, opts ...client.Option) (*AggsResponse, error) {
```

Comment:
making this name change for consistency. we're eventually going to have a `reference` package where the method names will need to include the resource type and i think it's important to be consistent across the repo